GraphTheory/Dijkstra.cpp

```diff
@@ -0,0 +1,55 @@
+#include <iostream>
+#include <vector>
+#include <queue>
+
```

Comment:
Add the "description comment" at the very top as mentioned [here](../blob/master/CONTRIBUTING.md#general-guidelines).